packages/block-library/src/columns/edit.native.js

```diff
@@ -69,110 +77,109 @@ const DEFAULT_COLUMNS_NUM = 2;
  */
 const MIN_COLUMNS_NUM = 1;
 
-/**
- * Maximum number of columns in a row
- *
- * @type {number}
- */
-const MAX_COLUMNS_NUM_IN_ROW = 3;
-
-const BREAKPOINTS = {
-	mobile: 480,
-	large: 768,
-};
-
 function ColumnsEditContainer( {
 	attributes,
 	updateAlignment,
 	updateColumns,
 	columnCount,
 	isSelected,
-	onAddNextColumn,
 	onDeleteBlock,
 	innerColumns,
 	updateInnerColumnWidth,
+	parentBlockAlignment,
+	editorSidebarOpened,
 } ) {
 	const [ resizeListener, sizes ] = useResizeObserver();
 	const [ columnsInRow, setColumnsInRow ] = useState( MIN_COLUMNS_NUM );
+	const [ tempWidth, setTempWidth ] = useState( 0 );
+	const screenWidth = Math.floor( Dimensions.get( 'window' ).width );
+	const { globalStyles } = useContext( GlobalStylesContext );
 
-	const { verticalAlignment } = attributes;
+	const { verticalAlignment, align } = attributes;
 	const { width } = sizes || {};
 
 	const newColumnCount = columnCount || DEFAULT_COLUMNS_NUM;
```

Comment:
It could be created inside the useEffect

Reply:
What can be created inside the `useEffect`?

Reply:
nvm, I see